src/mui/field/RichTextField.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow, render } from 'enzyme';
+import RichTextField, { stripTags, truncate } from './RichTextField';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+
+const muiTheme = getMuiTheme({
+    userAgent: false,
+});
+
+describe('stripTags', () => {
+    it('should strip HTML tags from input', () => {
+        assert.equal(stripTags('<h1>Hello world!</h1>'), 'Hello world!');
```

Comment:
could you add a test with embedded tags, and with an incorrect HTML (i.e. something like `<p>foo<p>`)?